integration-tests/src/test/java/gov/uk/address/api/stepdefinitions/AddressSteps.java

```diff
@@ -112,4 +162,14 @@ public void user_does_not_get_any_address() throws JsonProcessingException {
         assertTrue(list.isEmpty());
         assertEquals(200, this.testContext.getResponse().statusCode());
     }
+
+    @And("the SQS events are deleted from the queue")
```

Comment:
This is nice to have and will let us know if we have any cleanup issues like we've been seeing in NINo 👍